main/externalData/balances/processor/index.ts

```diff
@@ -117,13 +132,13 @@ function handleBalanceUpdate(
   store.addPopulatedChains(address.toLowerCase(), chains, toExpiryWindow[mode])
 }
 
-function handleCustomTokenUpdate(customTokens: Token[]) {
+export function handleCustomTokenUpdate(customTokens: Token[]) {
   log.debug('Handling custom token update', { customTokens })
 
-  const accounts = api.getAccounts()
+  const accounts = storeApi.getAccounts()
```

Comment:
seems like some places we're accessing the store via this typed api object and some places we're doing it directly. can we make this consistent?